### ISSUE TITLE
Fork the mullvad split-tunneling driver

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#2122]
If a user installs the mullvad VPN and the Mozilla one on windows, the 2 apps do not work properly because the split-tunneling driver is locked down in such a way that only one user-land process can talk to it at a time.

Faern (mullvad) suggests: "You will need to fork the driver and modify some identifiers in it so that when you install it it's treated as a separate driver from ours (but the functionality will be identical). This allows our drivers to co-exist in peace. We need you to do roughly the following:

```
#1 Perform changes that correspond to what is mentioned here. Some steps may not be applicable: https://github.com/OpenVPN/tap-windows6/blob/154d825af1d4e6f493b561fc6dd06a7d58ac2235/src/OemVista.inf.in#L19

#2 Change the device identifier and its symbolic link name defined here: https://github.com/mullvad/win-split-tunnel/blob/1176dd6fb4be0b24eff57fec1a08e9573df664a1/src/driverentry.cpp#L39

#3 Update batch script used for signing, and dependent cross certificate.

#4 Update installer to use branded device name and hardware ID.
```
"

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-1224)
┆Link To Issue: https://mozilla-hub.atlassian.net/browse/VPN-1224

**Comment by faern** [comment]
Please fix this asap! The driver only allows a single user space process to be connected to it at a time. So users having both your VPN app and our (Mullvad's) installed at the same time will get into trouble. One of the VPN apps will fail to talk to the driver and malfunction in one way or another.

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Basti (Sebastian) Streich I believe this is addressed by disabling split tunneling when Mullvad is installed, right? I realize this is not the most elegant solution but it’s pretty efficient. Should we call this addressed?

**Comment by data-sync-user** [comment]
➤ Basti (Sebastian) Streich commented:

No. As said before, this is just a band-aid. We still need to do that 🙂

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Sebastian Streich (as the person who worked on this before)  Adrienne Davenport (as the tech debt component lead), is this really a ‘high’ priority bug? By looking at where this sits in both of the component-backlogs, I’d reckon we are saying ‘No’ (and I agree with that). 

Since this only impacts users who have Mullvad on top of the Mozilla VPN, I’d place this as ‘Low’ (but I ack that I may not have the full picture of why this could be more valuable than what I think).

**Comment by data-sync-user** [comment]
➤ Sebastian Streich commented:

Mullvad has been asking us to do this for a long time … and we failed to deliver there. 

However, since we now disable the usage of the split-tunnel driver whenever mullvad or any other potential conflict is detected - maybe we should check in with them if this is still a pressing issue?

**Comment by faern** [comment]
Hello! I'm from Mullvad. We are good I believe. If you no longer compete for the usage of the driver, we don't have any issue on our side :+1:

Previously your usage of the same driver caused us another issue: You uninstalled the driver when your app was uninstalled. so a user having both our apps and then uninstalling yours would leave our app without a working driver. But we fixed this by not installing the kernel driver during the Mullvad app install, but rather loading it for every tunnel connection we initiated if it was not already loaded. So we are good now.

Ping @dlon
